src/Data/List/ZipList.purs

```diff
@@ -65,3 +65,8 @@ instance plusZipList :: Plus ZipList where
   empty = mempty
 
 instance alternativeZipList :: Alternative ZipList
+
+instance zipListIsNotBind
+  :: Fail "ZipList is not Bind. Any implementation would break the associativity law"
+  => Bind ZipList where
+    bind _ _ = mempty
```

Comment:
Can we use `unsafeCrashWith` to avoid confusion (in case anyone thinks this is runnable code)?